lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -169,18 +169,18 @@ <h3 class="course-title">
                           share_text = _("I'm taking {course_name} online with edX.org. Check it out!").format(course_name=course_overview.display_name_with_default)
                           query_params = urllib.urlencode((('u', facebook_share_url), ('quote', share_text.encode('utf-8')),))
                           facebook_url = 'https://www.facebook.com/sharer/sharer.php?{query}'.format(query=query_params)
+                          share_msg = ("Share {course_name} on Facebook").format(course_name=course_overview.display_name_with_default)
```

Comment:
I think you should wrap the text `Share {course_name} on Facebook` with gettext function, `_()`